styles/drop-down-list/_styles.scss

```diff
@@ -0,0 +1,60 @@
+//== drop-down-list
+//
+//## styles for drop-down-list
+
+.daptiv-drop-down-list {
+    display: inline-block;
+    font: $dropdownlist-font;
```

Comment:
Please switch to `include font-size-medium()` It will set the correct font weight for that size and the font is globally set so it is unnecessary to reset this everywhere.